tracing/src/main/java/com/palantir/tracing/TagTranslator.java

```diff
@@ -63,6 +63,10 @@ default boolean isEmpty(@SuppressWarnings("unused") S data) {
      * Tag adapter object which insulates the implementation of the underlying data structure from callers.
      */
     interface TagAdapter<T> {
+        /**
+         * Implementations should ignore calls with {@code null} key or value, however implementations aren't
```

Comment:
"tread" -> "treat"?

I'm a bit confused by the language here: I would suggest writing it from the perspective of the user. "If either key or value are null, no information will be recorded, instead of throwing an NPE"